sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -137,7 +137,8 @@ public EventData(String body) {
      * @throws NullPointerException if {@code message} is null.
      */
     EventData(Message message) {
-        Objects.requireNonNull(message, "'message' cannot be null.");
+        Objects.requireNonNull(message,
+            EventHubErrorCodeStrings.getErrorString(EventHubErrorCodeStrings.MESSAGE_CANNOT_NULL));
```

Comment:
Static import to make the lines shorter (so that they don't wrap). If you static import the getErrorString method and the EventHubErrorCodeStrings types, it would just be `getErrorString(MESSAGE_CANNOT_NULL)`.